docs/src/ice_nucleation_plots/KnopfAlpert2013_fig1.jl

```diff
@@ -0,0 +1,167 @@
+import Plots
+
+import CloudMicrophysics
+import CLIMAParameters
+import Thermodynamics
+
+const PL = Plots
+const IN = CloudMicrophysics.HetIceNucleation
+const CMP = CloudMicrophysics.Parameters
+const CT = CloudMicrophysics.CommonTypes
+const CO = CloudMicrophysics.Common
+const CP = CLIMAParameters
+const TD = Thermodynamics
+
+include(joinpath(pkgdir(CloudMicrophysics), "test", "create_parameters.jl"))
+FT = Float64
+toml_dict = CP.create_toml_dict(FT; dict_type = "alias")
+const prs = cloud_microphysics_parameters(toml_dict)
+thermo_params = CMP.thermodynamics_params(prs)
+
+# Initializing
+temp = collect(210.0:2:232.0) # air temperature
+x = 0.1                       # wt% sulphuric acid in droplets
+Delta_a = Vector{Float64}(undef, length(temp))
+J = Vector{Float64}(undef, length(temp))
+
+# Knopf and Alpert 2013 Figure 4A
+# https://doi.org/10.1039/C3FD00035D
+dust_type = CT.KaoliniteType()
+
+it = 1
+for T in temp
+    Delta_a[it] = CO.Delta_a_w(prs, x, T)
```

Comment:
Instead of doing a loop and dealing with allocations we could write it in a shorter way. Here is an example (without the paper data, just our line):

```
import Plots as PL                                                              
import CloudMicrophysics as CM                                                  
import CLIMAParameters as CP                                                    
                                                                                
const IN = CM.HetIceNucleation                                                  
const CT = CM.CommonTypes                                                       
const CO = CM.Common                                                            
                                                                                
include(joinpath(pkgdir(CM), "test", "create_parameters.jl"))                   
FT = Float64                                                                    
toml_dict = CP.create_toml_dict(FT; dict_type = "alias")                        
const prs = cloud_microphysics_parameters(toml_dict)                            
                                                                                
x = 0.1                                     # wt% sulphuric acid in droplets    
dust_type = CT.KaoliniteType()                                                  
T_range = range(210, stop=232, length=100)  # temperature                       
Δa_w = [CO.Delta_a_w(prs, x, T) for T in T_range]                               
J = @. IN.ABIFM_J(prs, (dust_type,), Δa_w)                                      
log10J_converted = @. log10(J * 1e-4)                                           
                                                                                
# Knopf and Alpert 2013 Figure 4A                                               
# https://doi.org/10.1039/C3FD00035D                                            
PL.plot(                                                                        
    Δa_w,                                                                       
    log10J_converted,                                                           
    label = "CliMA",                                                            
    xlabel = "Δa_w [-]",                                                        
    ylabel = "log_10(J) [cm^-2 s^-1]",                                          
)                                                                               
PL.savefig("Knopf_Alpert_fig_1.svg")  
```